### PR TITLE
Fix db scripts

### PR DESCRIPTION
**Pull request by pawar4**
Made changes in data_insert and ftp_transfer to make the db scripts work with how it was intergrated in main.py

### CODE REVIEW

File: src/db/data_insertion.py [hunk 1]
```diff
@@ -13,16 +18,22 @@ def data_insert(image_name: str, image_date: datetime, image_time: datetime, ini
...
         bboxes (list): list of bounding boxes, each bounding box containing coordinates, confidence and classification
+        labels (list): defines the classifications for each bounding box
         input_dir (string): image path in client machine
     """
+    
+    """The 3 classifications included in labels are represented as follows:
+    glasses : 0, goggles : 1, neither : 2
+    Glasses classification is currently not being stored in the database so the list is modified before database insertion
+    """
+    labels = [GOGGLES if label == 1 else NEITHER for label in labels]
```
Comment (stephend017) [inline src/db/data_insertion.py:29]: here we have 3 classifications which is inconsistent with what was defined above. also they have different values? this is resulting in unnecessary code like this if statement

File: src/db/data_insertion.py [hunk 2]
```diff
@@ -13,16 +18,22 @@ def data_insert(image_name: str, image_date: datetime, image_time: datetime, ini
...
 
     # Below ftp transfer has been commented out for testing purposes and will be fixed by another team member
```
Comment (stephend017) [inline src/db/data_insertion.py:31]: delete this comment too

**Comment by ZPBerg** [comment]
Perhaps we should just have all 3 classifications in the database, represented as 0, 1, 2. It's not hard to query later for a specific class and this would provide more information if we wanted to do analysis of the classification at a later time.

File: src/db/data_insertion.py [hunk 3]
```diff
@@ -3,8 +3,13 @@
 from decimal import Decimal
 import datetime
 
+"""Only neither and goggles classifications are currently stored in the database as a bit value. Hence these variables define
+the standard being used.
+"""
+GOGGLES = 1
+NEITHER = 0
```
[the remark this answers is not in the document]
Comment (pawar4) [reply src/db/data_insertion.py]: I am going to make the change that Zach suggested and store all classifications (goggles, glasses, neither) in the database. If I do this then I will not have to do any sort of processing of the classifications. I just insert classification as is to the database. Given this, do I really need to make this an enum here? I would think having this an enum would be more useful in a query script

[the remark this answers is not in the document]
Comment (stephend017) [reply src/db/data_insertion.py]: So the reason for an enum is focused more around context, and maintainability. if we have an enum then we get that implicit error checking built in for us. if we leave it as an `int` then we could potentially have a change that corrupts our data. especially if we expanded the potential values. 

the big takeaways from using an enum is maintainability and documentation. the enum is easy to extend, makes our code harder to break and doesn't require new documentation to be written when its modified.

**Pull request merged**